app/retail/emails.py

```diff
@@ -1135,7 +1178,7 @@ def weekly_recap(request):
 
 @staff_member_required
 def unread_notification_email_weekly_roundup(request):
-    response_html, _ = render_unread_notification_email_weekly_roundup('mark.beacom@consensys.net')
+    response_html, _, _ = render_unread_notification_email_weekly_roundup('sebastian.tharakan97@gmail.com')
```

Comment:
Should we render this email such that we use the requester's email rather than hardcode it?

Reply:
The hardcoded email was actually already there:https://github.com/gitcoinco/web/blob/144365cfa682509b38860bab6bec4b86c6dc98ae/app/retail/emails.py#L1138
`response_html, _, _ ` is what I actually wanted to add because it's part of the bugfix above. See commit [18d165d](https://github.com/gitcoinco/web/pull/6471/commits/18d165dd28d86dfc58f946ea6b7ef22fc093ab70)

Could change it to:
```
response_html, _, _ = render_unread_notification_email_weekly_roundup(request.user.email)
```
 Should I?